sycl/test-e2e/Matrix/element_wise_abc.cpp

```diff
@@ -10,6 +10,9 @@
 // RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4
 // RUN: %{run} %t.out
 
+#include <cstddef>
+
 #define SG_SZ 16
```

Comment:
I thought we were getting rid of this macro in all tests?

Reply:
This is not subject of this PR. I don't want to mix refactoring with adding tests for SG=32. in all new code that I add, I used constexpr, but I did not touch the code that I did not intend to change.